apps/front-end/front-office/src/app/reducers/game/game.models.ts

```diff
@@ -6,7 +6,11 @@ import { Patient, Quiz } from '@webonjour/util-interface';
 export interface GameEntity {
   quiz: Quiz.Quiz | null;
   score: number;
-  currentQuestion: number;
   times: number[];
```

Comment:
remove times and score, wrong questions, remaning questions
Use a new type for history containing this information
```
interface HistoryItem{
    questionId: number,
    isCorrect: number,
    timeTaken: number,
```

modify selectors to get the wrong questions by filtering over the ids in history
same for the other fields

Reply:
On it

Reply:
Well, you were faster than me